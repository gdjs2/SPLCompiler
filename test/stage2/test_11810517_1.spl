int x;

int f(int a, int b) {
	int y;
	a = 10;
	b = a;
	z = 10;
}