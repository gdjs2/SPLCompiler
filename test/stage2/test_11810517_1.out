Error type 1 at Line 7: variable "z" is used without definition.
Error type 5 at Line 7: unmatching types appear at both sides of the assignment operator